src/components/HeadCellMenu/HeadCellMenu.tsx

```diff
@@ -18,25 +18,19 @@ const HeadCellMenu = ({
   searchRelatedArgs,
   selectionRelatedArgs,
   adjustHeaderSizeRelatedArgs,
+
+  openMenuDropdown,
```

Comment:
I got a bit confused by the naming here. Thinking that it was a callback function and not a boolean value that indicates if the menu is opened or not.

Buuuut I think it's fine 👍

Reply:
I've changed it to simple `open` and `setOpen` since the entire dropdown is not controlled by the repo where it's been used.